Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/magnum.yml

```
@@ -41,7 +41,7 @@
   - type: Projectile
     damage:
       types:
-        Piercing: 25
+        Piercing: 20 #WWDP edit tungsten core you know?
```

```suggestion
        Piercing: 35 #WWDP edit
```
Бронепробивные редкие патроны. Их не нужно дебафать ибо их получение теперь только через аплинк

---

не только,их можно вырастить в ботанике

---

Те что в ботанике вроде без боевых, но если и правда с ними, то лучше проста заменить, а не портить аплинк айтем

---

окей